src/components/index.ts

```diff
@@ -222,8 +222,8 @@ export {default as ScrollLock} from './ScrollLock';
 export {
   default as Select,
   Props as SelectProps,
-  Option as SelectOption,
-  Group as SelectGroup,
+  SelectOption,
```

Comment:
As we're not changing the export names in `src/components/index.ts` this is **not** a breaking change to our public API